Fleece/MSVC_Compat.hh

```diff
@@ -38,6 +38,10 @@
     // WARNING: sizeof() will not work on this array since it's actually declared as a pointer.
     #define StackArray(NAME, TYPE, SIZE)    TYPE* NAME = (TYPE*)_malloca(sizeof(TYPE)*(SIZE))
 
+    #define strlcat strncat
```

Comment:
Hmmm I didn't know a lot about this.  However, will this still work in the presence of non-ASCII UTF-8 characters?  If not, should a function be written to handle that?